quic/s2n-quic-crypto/Cargo.toml

```diff
@@ -22,7 +22,13 @@ s2n-quic-core = { version = "=0.32.0", path = "../s2n-quic-core", default-featur
 zeroize = { version = "1", default-features = false, features = ["derive"] }
 
 [target.'cfg(not(target_os = "windows"))'.dependencies]
-aws-lc-rs = { version = "1.5" }
+# Restrict the version to avoid symbol conflicts from multiple version of 'aws-lc-sys'.
+#
+# Both 's2n-quic-crypto' and 's2n-tls-sys' end up depending on 'aws-lc-sys'. Update
+# this dependency once s2n-tls-sys starts consuming a higher version of aws-lc-sys.
+#
+# https://github.com/aws/s2n-quic/pull/2104#issuecomment-1918559366
+aws-lc-rs = { version = "=1.5" }
```

Comment:
This isn't great. It will require us to do the same thing every time aws-lc-sys gets a new minor release. I think either aws-lc-sys also needs to bump to the same major version as aws-lc-rs or aws-lc-rs needs to export the C headers/lib dir so s2n-tls-sys can use it.

Reply:
While you're here, we should also do `default-features = false`

Reply:
If you do `default-features = false`, you  will need to add `features = ["aws-lc-sys"]`.  

(Consumers will still be able to select the "fips" feature with this. If you use the "non-fips" feature, then "fips" is blocked.)

Reply:
Although, thinking about that a bit more, it does mean that if you enable fips that both sys and fips-sys will be built, since we've forced that dependency here. Not sure if there's a way around that, though.